src/expr/src/linear.rs

```diff
@@ -1882,140 +1881,6 @@ pub mod plan {
                 || self.upper_bounds.iter().any(|e| e.could_error())
         }
     }
-
-    /// Logic for turning an [MfpPlan] into filters that can confidently skip
-    /// fetching an entire persist part based on aggregate statistics (e.g.
-    /// min/max) about the data contained in it.
-    #[derive(Debug)]
-    pub struct MfpPushdown {
```

Comment:
Why is this no longer needed?

Reply:
This code was effectively doing a hard-coded subset of the abstract-interpretation problem that is now solved in general! There's still a little code that translates between persist stats and `ResultSpec`s, which now lives in the Persist source.